src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/ReportableTaskController.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.response.ReportableTaskResponse;
+import uk.gov.hmcts.reform.wataskmanagementapi.services.MIReportingService;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@Profile("replica | preview")
+public class ReportableTaskController {
+    private final MIReportingService taskHistoryService;
+
+    public ReportableTaskController(MIReportingService taskHistoryService) {
+        this.taskHistoryService = taskHistoryService;
+    }
+
+    @GetMapping(
+        path = "/task/{task-id}/reportable",
+        produces = APPLICATION_JSON_VALUE
+    )
+    public ReportableTaskResponse getReportableTask(@PathVariable("task-id") String taskId) {
```

Comment:
Can anyone look into the task data? Shouldn't this have same authorisation as get task endpoint?

Reply:
Its only there for testing, we don't actually need it in prod.